.github/workflows/ci.yml

```
@@ -5,11 +5,9 @@ on:
     branches: [ 'dblp' ]
   pull_request:
     branches: [ 'dblp' ]
-  issue_comment:
-    types: [ 'created', 'edited' ]
 
```

We just realize that the build job needs the GITHUB_TOKEN to fetch languages !!! And it's missing in the files.
Could you add this few lines 🙂
```suggestion
env:
  GIT_TOKEN: ${{ secrets.GIT_TOKEN }}
```